### PR TITLE
[Code cleanup] Remove unused $stationery_path parameter

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Remove $stationery_path which was used for an old pdf package but is no longer used

Before
----------------------------------------
lines of code to do nothing

After
----------------------------------------
lines removed

Technical Details
----------------------------------------
cleanup only

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Utils/PDF/Utils.php [hunk 1]
```diff
@@ -98,23 +90,23 @@ public static function html2pdf(&$text, $fileName = 'civicrm.pdf', $output = FAL
   <head>
     <meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\"/>
     <style>@page { margin: {$t}{$metric} {$r}{$metric} {$b}{$metric} {$l}{$metric}; }</style>
-    <style type=\"text/css\">@import url({$config->userFrameworkResourceURL}css/print.css);</style>
+    <style type=\"text/css\">@import url(" . CRM_Core_Config::singleton()->userFrameworkResourceURL . "css/print.css);</style>
```
Comment (colemanw) [inline CRM/Utils/PDF/Utils.php]: What's the reason for this change? Looks to me like the `$config` variable has been defined and should work fine here.

Comment (eileenmcnaughton) [reply CRM/Utils/PDF/Utils.php]: yeah it does but it doesn't really need to be defined since the other place it's being used should really use a setting - using config instead of setting is legacy - and it should really be in a separate fn anyway so no need to declare $config after all
